src/app/screens/Settings.tsx

```diff
@@ -140,6 +157,26 @@ function Settings() {
             <LocaleSwitcher />
           </div>
         </Setting>
+
+        <Setting title="Theme" subtitle="Change the app theme to dark or light">
+          {!loading && (
+            <div className="w-64">
+              <Select
+                name="theme"
+                value={settings.theme}
+                onChange={(ev) => {
+                  saveSetting({
+                    theme: ev.target.value,
+                  });
+                }}
+              >
+                <option value="dark">Dark</option>
```

Comment:
Now that we have translaitons in should each ne PR already be forced to use translation-placeholders instead of strings?

Reply:
Ok I can update this with the translations.

Reply:
I just noticed none of the other items on this page have the translations yet it looks like, should we put in a separate PR to update the entire app with translations?

Reply:
Created #781 for that.  
But it might be good to just start with this and do it step by step already.